app/assets/javascripts/help.js

```diff
@@ -1,13 +1,28 @@
 // Hides and shows each page of Help
 
-function open_window(id) {
+function openWindow(id, parentId) {
```

Comment:
This case is also fixed in this open [PR](https://github.com/simpledotorg/simple-server/pull/1221), but it should be a clean merge regardless.